src/platform/forms-system/test/js/utilities/escapeRegExp.unit.spec.js

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
```

Comment:
Could you add a test here for 'should not escape regular expression characters that are not special or escaped'? Just to cover the opposite scenario.

Reply:
Done. I also added a step where the string is used to create a RegExp to ensure that an error isn't thrown. Hmm, I'm not sure if I need to explicitly add a try/catch around the tests and add a `not.thrown` check...

Reply:
I'm not sure that's necessary, at least not for this function. It doesn't return a promise, just escapes characters.